.github/workflows/update-ref.yml

```diff
@@ -0,0 +1,56 @@
+name: Update references for commits, tags, etc
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 1 * *' # Run on the first day of every month
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        shell: bash
+
+    env:
+      DBUS_SESSION_BUS_ADDRESS: unix:path=/run/user/1001/bus
+      SHELL: /usr/bin/bash
+
+    strategy:
+      matrix:
+        node-version: [20]
+
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v3
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Update required extension version
+      run: bash scripts/update-extension-version.sh
+
+    - name: Update fetch-docs script
+      run: bash scripts/update-fetch-docs-commits.sh
+
+    - name: Verify file changes
+      uses: tj-actions/verify-changed-files@v17
+      id: verify-changed-files
+      with:
+        files: |
```

Comment:
This should be true if any of them change (not just when both changed). Is it correct?

Reply:
Yes